File: sample-videochat-webrtc/sample-videochat-webrtc/ViewControllers/Settings/VideoSettingsViewController/BaseItemModel.m
//
//  BaseItemModel.m
//  sample-videochat-webrtc
//
//  Created by Andrey Ivanov on 30/09/15.
//  Copyright © 2015 QuickBlox Team. All rights reserved.
//

#import "BaseItemModel.h"
#import "SettingCell.h"

@implementation BaseItemModel

- (instancetype)initWithTitle:(NSString *)title data:(id)data {
    self = [super init];
    if (self) {
        self.title = title;
        self.data = data;
    }
    return self;
}

- (instancetype)initWithTitle:(NSString *)title {
    return [self initWithTitle:title data:nil];
}


- (Class)viewClass {
    
    return [SettingCell class];
}

@end

File: sample-conference-videochat-swift/sample-conference-videochat-swift/sample-conference-videochat-swift/Components/Settings/BaseSettingsViewController/BaseItemModel.swift
//
//  BaseItemModel.swift
//  sample-conference-videochat-swift
//
//  Created by Vladimir Nybozhinsky on 22.10.2018.
//  Copyright © 2018 QuickBlox. All rights reserved.
//

import UIKit

class BaseItemModel {
    var title = ""
    var data: Any?
    
    func viewClass() -> AnyClass {
        return  SettingCell.self
    }
}
